fix EWS split header error (#10939)
demisto/content: Packs/EWS/ReleaseNotes/1_7_3.md

#### Integrations
##### EWS v2

> - Fixed an issue with ***fetch incidents***, was failed to parse some emails headers.

Reviewer: ```suggestion
Fixed an issue where ***fetch incidents*** failed to parse the email headers.
```
